app/Libraries/SSO/RsaSha1.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+
+namespace App\Libraries\SSO;
+
+use Eher\OAuth\OAuthException;
+use Eher\OAuth\RsaSha1 as BaseRsaSha1;
+
+class RsaSha1 extends BaseRsaSha1
+{
+    private $cert = false;
+
+    public function __construct($cert)
+    {
+        $this->cert = $cert;
+    }
+
+    /**
+     * Fetch the private RSA certificate
+     *
+     * @param $request
+     * @return string
+     */
+    public function fetch_private_cert(&$request)
+    {
+        return $this->cert;
+    }
+
+    /**
+     * Fetch the public RSA certificate (we do not use this functionality)
+     *
+     * @param $request
+     * @throws OAuthException
+     */
+    public function fetch_public_cert(&$request)
```

Comment:
Are these methods meant to be snake_cake? We have used camelCase elsewhere.

Reply:
Required because that is how the base class is 🤷‍♂️

Reply:
I knew that 🤣